### PR TITLE
Support -N option in tachyon-start.sh

### PR DESCRIPTION
**Pull request by chu11**
Presently, starting master and workers in tachyon-start.sh (via 'all' or 'local')
well check for and attempt to kill previously running masters and/or
workers.  This is an excessive check in some environments.

This patch supports a -k option to not check for and kill previously
running processes.

### CODE REVIEW

File: bin/tachyon-start.sh [hunk 1]
```diff
@@ -26,6 +26,8 @@ MOPT is one of:
 
 -f  format Journal, UnderFS Data and Workers Folder on master
 
+-k  Do not try to kill prior running masters and/or workers in all or local
```
Comment (calvinjia) [inline bin/tachyon-start.sh]: -k seems to imply wanting to kill the running processes? Perhaps something like -no-kill, though its more verbose.

Comment (chu11) [inline bin/tachyon-start.sh]: How about 'N'?  I kept it to single characters b/c the script is currently using the bash builtin of 'getopts', which (AFAIK) doesn't support long args.

**Comment by chu11** [comment]
Updated patch to use 'N' instead of 'k' as option.

File: bin/tachyon-start.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ fi
 
 #start up tachyon
 
-Usage="Usage: tachyon-start.sh [-h] WHAT [MOPT] [-f]
+Usage="Usage: tachyon-start.sh [-hk] WHAT [MOPT] [-f]
```
Comment (calvinjia) [inline bin/tachyon-start.sh]: does this need to be -hN?

Comment (chu11) [inline bin/tachyon-start.sh]: Doh!  Thanks for the catch, branch updated

**Comment by calvinjia** [comment]
+1 LGTM

**Comment by haoyuan** [comment]
Merged. 

@chu11 Are you running multiple Tachyon systems in the same cluster?

**Comment by chu11** [comment]
Not yet, but that is the plan.  I am integrating Tachyon into my project Magpie (https://github.com/chu11/magpie), which allows users to run an assortment of "Big Data" projects on the same cluster.

**Pull request merged**